test/helpers.js

```diff
@@ -0,0 +1,30 @@
+// Helper function to determine whether this browser is supported by 
+// impress.js or not. Copied from impress.js itself.
+var _impressSupported = function() {
```

Comment:
By the way, I was convinced about the point of not touching impress code, but I still don't see a need for the underscore as I commented above:

> The current code is not doing any encapsulation, it is exposed as a global variable with or without underscore. There is no difference between prefixing with underscore or not, in either case we need to take care to acknowledge the globals that are being used in testing and take care for not overriding it.